src/components/ThemeProvider/ReactIcons.tsx

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable react-refresh/only-export-components */
```

Comment:
is this something we would want to do in other file as well or can we resolve this lint issue by only exporting components?